src/api/plugin.ts

```diff
@@ -21,6 +21,7 @@ const createResponseInterceptor = (store) => {
     if (status === 401) {
       store.dispatch('auth/logout')
     }
+    throw error
```

Comment:
Just wondering: why the lonely error here?

Reply:
I found out that there was a problem when the interceptor did not throw the error, it was "consumed" by it and was not propagated so it cannot be handled in the code where the actual request was made.